crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -496,11 +530,30 @@ impl<'src> Parser<'src> {
         let start = self.node_start();
         self.bump(TokenKind::Nonlocal);
 
+        // test_err nonlocal_stmt_trailing_comma
+        // nonlocal ,
+        // nonlocal x,
+        // nonlocal x, y,
+
+        // test_err nonlocal_stmt_expression
+        // nonlocal x + 1
```

Comment:
Yeah that's a good idea. I mean usually editor would allow custom query to highlight certain part of the code in a different way. I know Neovim allows it with treesitter ;)